HomeUI/src/views/apps/marketplace/AppView.vue

```diff
@@ -796,7 +885,11 @@ export default {
         }
         if (props.appData.version >= 7) {
           appSpecification.staticip = props.appData.staticip;
-          appSpecification.nodes = [];
+          if (props.appData.compose?.some((c) => (c?.enterpriseApp === true))) {
```

Comment:
I think this should not be in compose. 
enterpriseApp is more global affecting entire application, so it should be just props.appData.enterpriseApp === true.
Then we need to be sure that enterpriseApp flag does not get it to specifications but only acts for ent. nodes selection

Reply:
Added the changes based on what I wrote above. Seems like the more plausible route. Please review when you get a chance @TheTrunk